data-generator/generate_data_test.py

```diff
@@ -205,16 +200,21 @@ def test_request(valid_change_request, make_change_request):
 def test_all_changesets_linked_with_before_attribute(generate_all_changesets):
     all_changesets = generate_all_changesets
     for i in range(1, len(all_changesets)):
-        prev_change_sha = (all_changesets[i - 1].get("checkout_sha")
-                           or all_changesets[i - 1].get("head_commit", {}).get("id"))
+        prev_change_sha = all_changesets[i - 1].get("checkout_sha") or all_changesets[
+            i - 1
+        ].get("head_commit", {}).get("id")
```

Comment:
Not a fan of how `ruff` formats these `or` expressions, but we can fix that manually by using additional local variables.

See also [data-generator/generate_data.py](https://github.com/mozilla-services/fourkeys/pull/77/files#diff-a4086723f26c48922af02e59c87620eee44ed27a76c3dc8c775983d871e3da15)

Reply:
[d536b05](https://github.com/mozilla-services/fourkeys/pull/77/commits/d536b0558c27aa6e26ebb614bbe7aebf0a523c13)